### PR TITLE
update embed event docs to include new voter data

### PR DESCRIPTION
**Pull request by ebehlmann**
Updating documentation on the JS events fired by embed codes to reflect the fact that new properties about the voter have been added (first_name, last_name, zipcode, email)

[Ticket 7935](https://app.shortcut.com/turnout2020/story/7935/add-more-details-to-js-events)

### CODE REVIEW

File: docs/embed/gtm.md [hunk 1]
```diff
@@ -128,13 +164,6 @@ Here's a list of events provided by VoteAmerica. Some events have variable param
   state: '[STATE]',
 }
 
-{
-  event: 'action-finish',
-  method: 'pdf',
-  tool: 'absentee',
-  state: '[STATE]',
-}
-
```
Comment (TildeWill) [inline docs/embed/gtm.md:137]: Did this turn out to not be a real event? I'm curious why it got deleted.

**Review by TildeWill — APPROVED** [review]
Had a question about the event that looks like it is getting removed form the docs, otherwise this looks good to me. Thanks for digging through this!

File: docs/embed/gtm.md (hunk 1 above)
Comment (ebehlmann) [reply docs/embed/gtm.md:137]: I should probably run this by you actually. 

But in the docs, this was listed twice, so I was really just removing a duplicate: 
```
{
  event: 'action-finish',
  method: 'pdf',
  tool: 'absentee',
  state: '[STATE]',
}
```

But I think the reason it was duplicated is that there are 2 JS files that produce this event - `GetMyForm.js` and `SignMyForm.js`. I was able to find and update the `GetMyForm` one, but I haven't figured out what triggers the other.

**Comment by TildeWill** [comment]
Great find, kill it!

**Pull request merged**